docs/iris/src/developers_guide/code_format.rst

```diff
@@ -0,0 +1,33 @@
+.. _iris_code_format:
+
+Code Formatting
+***************
+
+Iris uses `black <https://black.readthedocs.io/en/stable/>`_ formatting and `flake8 <https://flake8.pycqa.org/en/stable/>`_
+linting to enforce a consistent code format throughout the project. ``black`` and ``flake8`` can easily be installed
+into your development environment with ``pip``::
```

Comment:
It might be worthwhile mentioning that this is also possible (and equivalent) with conda.